ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/ChatCommandsPopup.qml

```diff
@@ -58,21 +68,30 @@ Popup {
             rotatedImage: true
             //% "Request transaction"
             text: qsTrId("request-transaction")
+            onClicked: function () {
+                chatCommandModal.commandTitle = qsTr("Request")
+                chatCommandModal.title = chatCommandModal.commandTitle
+                chatCommandModal.finalButtonLabel = qsTr("Request")
+                chatCommandModal.selectedRecipient = {
+                    address: Constants.zeroAddress, // Setting as zero address since we don't have the address yet
+                    identicon: chatsModel.activeChannel.identicon,
+                    name: chatsModel.activeChannel.name,
+                    type: RecipientSelector.Type.Contact
+                }
+                chatCommandModal.sendChatCommand = function(address, amount, tokenAddress) {
```

Comment:
Is there a particular reason this handler is created during the `onClicked` event? Would it be better if it was put directly in to the `ChatCommandModal` declaration?
```
ChatCommandModal {
    id: chatCommandModal
    sendChatCommand: function(address, amount, tokenAddress) {
        chatsModel.requestTransaction(chatsModel.activeChannel.id,
                                                  address,
                                                  amount,
                                                  tokenAddress)
         chatCommandModal.close()
     }
}
```